CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Support for `ipfs`/`ipns` URLs
 - Mode field for regex hint bindings
+- Reload configuration on SIGUSR1
```

Comment:
I'd be curious where else this feature can/needs to be advertised. I don't think we should just expect people to figure it out on their own.

Maybe we could document the live config reload in our features.md and then add the ability to manually trigger a reload to that?

Reply:
Done.